### ISSUE TITLE
[TW-530] Dependencies not documented

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#565]
_Marek Vitek on 2012-04-13T09:39:56Z says:_ 

 There is missing documentation on how to enter information that task is dependent on another task.
I tried to find it either by running command (version 1.9.4):
task help
or checking quick reference document for version 1.9.x and 2.0.0.

The only place with information about dependencies is tutorial.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-04-13T09:39:56Z
Modified: 2014-02-09T02:04:40Z
```

**Comment by taskwarrior** [comment]
_Dirk Deimeke on 2012-04-13T09:52:47Z says:_ 

 (at)man task@ works as well.

<pre>
depends:<id1,id2 ...>
    Declares  this  task  to  be  dependent on id1 and id2.  This means that the tasks id1 and id2
    should be completed before this task.  Consequently, this  task  will  then  show  up  on  the
    'blocked' report.
</pre>

But you are right, (at)task help@ does not show information about the modifier and it is missing on the "Cheat Sheet" as well.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-07-09T05:58:00Z says:_ 

 Dependencies are mentioned in task.1, are detailed in the wiki, and shown in the video and man page tutorials.  I think we're good.  I have made a note to include dependencies (and waiting) in the next release of the PDF cheat sheet.  When the new 2.x tutorial is ready, it will include an expanded description also.  Thanks for pointing this out Marek, your name is in the AUTHORS file.
